### ISSUE TITLE
Unable to find template plugin matching 'introspection'

### ISSUE THREAD

**Issue by Fi1osof** [dotansimha/graphql-code-generator#4899]
**Describe the bug**
 
Related to https://github.com/dotansimha/graphql-code-generator/issues/3769

**To Reproduce**

```bash
# clone repo
# yarn.lock exists
git clone https://github.com/prisma-cms/prisma-cms.com/ && cd prisma-cms.com

# checkout branch graphql-codegen-error
git checkout graphql-codegen-error

# install deps
yarn --ignore-engines

# run script 
yarn generate:types
```

<!--
#############
#############
🛑 STOP! 🛑 STOP! 🛑 STOP! 🛑 STOP! 🛑 STOP! 🛑 STOP! 🛑 STOP! 🛑 STOP! 🛑 STOP! 🛑 STOP! 🛑 STOP! 

PLEASE READ THE FOLLOWING: 
You can use this template to create a live sandbox of the issue: 
https://codesandbox.io/s/github/dotansimha/graphql-code-generator-issue-sandbox-template
Make sure sure fork this template, and run `yarn generate` in the terminal. 
Please make sure the Codegen and plugins version under `package.json` matches yours. 
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
~~~~~~~~~ ISSUES WITH A LIVE REPRODUCTION WILL BE ANSWERED QUICKLY ~~~~~~~
~~~~~~ ISSUES WITHOUT REPRODUCTION MIGHT GET CLOSED WITHOUT RESPONSE ~~~~~
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
#############
#############
-->

P.S. Can not run codesandbox due 502 error

1. My GraphQL schema:

```graphql
# Put your schema here
```
https://api.prisma-cms.com/

2. My GraphQL operations:

```graphql
# Put your operations here
```

**Expected behavior**
<!-- A clear and concise description of what you expected to happen. -->

Got from https://github.com/prisma-cms/nextjs where all works fine.
You may got this and run 
```bash
API_ENDPOINT=https://api.prisma-cms.com/ yarn generate:types
```

**Environment:**

- OS: Ubintu-20
- `@graphql-codegen/...`:  latest
- NodeJS: v12.19.0

**Additional context**
<!-- Add any other context about the problem here. -->

**Comment by Fi1osof** [comment]
It's realy brainpain...

Please, do publish new version in npm registry. Seems like this bugfix already exists in new commits, maybe https://github.com/dotansimha/graphql-code-generator/pull/4869/files
but not published yet.

Reason: i clone https://github.com/dotansimha/graphql-code-generator and build localy, do yarn link graphql here and in my project, and yarn link @graphql-codegen/cli in my project. Run yarn generate:types executed success.
Then i do git checkout 1422ab122c8d40cd7ec81c66ec67b7bf3d88527e -b test && yarn build (build commit, published to npmjs) and execution got this error.

Also in my project i edit node_modules/@graphql-codegen/cli/index.cjs.js and replace possibleNames with just   possibleNames = [
        `@graphql-codegen/introspection`,
    ];

And still error.

Finaly i add rreturn require('@graphql-codegen/introspection'); and this works!

**Comment by dotansimha** [comment]
@Fi1osof the original issue you mentioned is from Apr 2. It has nothing to do with this specific bug, so I locked it to avoid confusion, the exact same confusion that leads people to think this was related to the issue we had now, and exact same confusion that causes people to think open-source maintainers works for them.

**Comment by dotansimha** [comment]
Fixed in `@graphql-codegen/cli@1.18.0`
